packages/components/src/components/DistributionsChart/index.tsx

```diff
@@ -44,7 +46,15 @@ const InnerDistributionsChart: FC<{
   samples: number[];
   height: number;
   plot: SqDistributionsPlot;
-}> = ({ shapes, samples, plot, height: innerHeight, isMulti }) => {
+  showSamplesBar: boolean;
```

Comment:
Later in this component, there's `samplesFooterHeight` that's based on samples array being empty.

This makes it render a 10px footer even if we don't show the samples bar.

I'd suggest removing this prop, it's enough to pass `samples={[]}` to this component if we don't want to render the footer.

Reply:
I refactored it to hide that footer, but I still prefer passing in this boolean flag, for reasons of semantics.